staging/publishing/rules.yaml

```diff
@@ -614,21 +614,22 @@ rules:
       dir: staging/src/k8s.io/kubectl
     name: master
     dependencies:
-    - repository: api
-      branch: master
-    - repository: apimachinery
-      branch: master
-    - repository: client-go
-      branch: master
+      - repository: api
```

Comment:
The changes for this file need to be reverted, but looks good otherwise. :smile:

Reply:
thanks, I rebased too early, I've restored/fixed this.